website/index.html

```diff
@@ -704,21 +704,21 @@
 
           <div class="d-flex justify-content-between">
             <div class="col-md-4 col-xs-4">
-              <p class="time"></p>
+              <p class="time">2 PM</p>
```

Comment:
Removing the 2 PM here makes the layout a little cleaner. We did this last semester so there aren't redundant times listed (2PM twice)

Reply:
Yeah I know we removed it, but I looked it over and until we can get the new layout I think I like making the times as clear as possible